src/libraries/DIRC/DDIRCGeometry.cc

```diff
@@ -57,8 +57,12 @@ DDIRCGeometry::Initialize(int runnumber) {
 	all_found &= jgeom->Get("//trd[@name='OWDG']/@Xmp_Ymp_Z", OWDG_XYZ);
 	
         if( !all_found ){
-                jerr << "Problem finding all DIRC geometry elements. Please make sure your HDDS is up to date!" << endl;
-                _exit(-1);
+				// Not all experiments use the DIRC (e.g. CPP) and so their geometries do not
```

Comment:
DGeometry handles loading the DIRC or not based on the CCDB table DIRC/install_status. (1/0 for installed/not-installed).  I think it makes sense to keep this as a fatal error as a sanity check in that case.  @jrstevenjlab should probably weigh in

Reply:
That's correct, this was agreed by @rjones30 to have a common flag for sim and recon on the install status for the DIRC.  Thanks to @sdobbs for adding the recent CCDB entries for the PrimEx and SRC runs.  

If people are running simulations with run numbers for future run periods (where the DIRC is currently on by default), then we could set the DIRC/install_status to be off in CCDB for future run periods by default until we collect that data.